applications/nrf5340_audio/src/modules/led.c

```diff
@@ -31,19 +28,12 @@ LOG_MODULE_REGISTER(led, CONFIG_LOG_LED_LEVEL);
 #define CORE_APP_STR "CORE_APP"
 #define CORE_NET_STR "CORE_NET"
 
-#define LABEL_AND_COMMA(node_id) DT_LABEL(node_id),
-#define GPIO_AND_COMMA(node_id) DT_GPIO_PIN(node_id, gpios),
-#define GPIO_LABEL_AND_COMMMA(node_id) DT_GPIO_LABEL(node_id, gpios),
-#define GPIO_FLAGS_AND_COMMMA(node_id) DT_GPIO_FLAGS(node_id, gpios),
+#define GPIO_DT_SPEC_GET_AND_COMMA(node_id) GPIO_DT_SPEC_GET(node_id),
```

Comment:
```suggestion
#define GPIO_DT_SPEC_GET_AND_COMMA(node_id) GPIO_DT_SPEC_GET(node_id, gpios),
```

I don't see how the label is used anymore. The label was used to get what core the gpio should be assigned to, as well as telling if it's a mono or color LED

Reply:
When I try to build this (after adding gpios as stated above) I get:
<err> led: No match for led unit. The dts is likely not properly formatted
<err> main: ERR_CHK Err_code: [-6] @ line: 147

Reply:
added back the `led_labels`. I realized that code is parsing LED labels at runtime to determine things like its unit or core. We need to fix that by describing things properly in DT, labels are not meant to store information but just to give a name (useful e.g. to debug).

I'll open an issue to fix that in a follow-up PR.